environment-creator-application/src/main/java/com/ca/apim/gateway/cagatewayconfig/Main.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Main {
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
+
+    public static void main(String[] args) {
+        System.getenv().forEach((k, v) -> logger.log(Level.INFO, k + "::" + v));
```

Comment:
this is not finished right?

Reply:
yes. This PR is just meant to introduce the Jar. My next one will have it build the environment bundle. Splitting them up so that it is easier to manage